cmd/hook.go

```diff
@@ -705,7 +705,8 @@ Gitea or set your environment appropriately.`, "")
 
 	if err == nil {
 		for _, res := range resp.Results {
-			hookPrintResult(res.ShouldShowMessage, res.IsCreatePR, res.HeadBranch, res.URL)
+			// AGit always creates a pull request so there is no point in prompting user to create one
+			hookPrintResult(res.ShouldShowMessage, false, res.HeadBranch, res.URL)
```

Comment:
But the backend may return true or false, why should we ignore the returned value? Maybe we should change the router level.